code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -968,13 +968,13 @@
 
 /obj/item/clothing/suit/flakjack
 	name = "flak jacket"
-	desc = "(I) A dilapidated jacket made of a supposedly bullet-proof material (Hint: It isn't.). Smells faintly of napalm."
+	desc = "(I) A dilapidated jacket made of ballistic nylon. Smells faintly of napalm."
```

Comment:
```suggestion
	desc = "(II) A dilapidated jacket made of ballistic nylon. Smells faintly of napalm."
```

The description should always match the coded tier value. Armor undergoing a refactoring anyways but while we still have linearmor thats the format.